$OpenBSD: patch-data_meson_build,v 1.1 2022/01/13 08:06:39 ajacoutot Exp $

data/meson.build:8:5: ERROR: Function does not take positional arguments.

Index: data/meson.build
--- data/meson.build.orig
+++ data/meson.build
@@ -6,7 +6,6 @@ desktop_config = configuration_data()
 desktop_config.set('icon', application_id)
 desktop_config.set('binary', gitg_name)
 i18n.merge_file(
-  desktop,
   type: 'desktop',
   input: configure_file(
     input: desktop + '.in.in',
@@ -42,7 +41,6 @@ appdata_config = configuration_data()
 appdata_config.set('app-id', application_id)
 appdata_config.set('gettext', gitg_name)
 i18n.merge_file(
-  appdata,
   type: 'xml',
   input: configure_file(
     input: appdata + '.in.in',
